src/languages/php.coffee

```diff
@@ -44,6 +44,12 @@ module.exports = {
       type: 'string'
       default: ""
       description: "Add rule(s). i.e. line_ending,-full_opening_tag,@PSR2 (PHP-CS-Fixer 2 only)"
+    allow_risky:
```

Comment:
Great! Don't forget to regenerate the documentation with `npm run docs`.

See https://github.com/Glavin001/atom-beautify/blob/master/docs/add-languages-and-beautifiers.md#options-documentation for details.